src/plugins/intel_gpu/src/kernel_selector/kernels/eltwise/eltwise_kernel_base.cpp

```diff
@@ -80,6 +80,14 @@ ParamsKey eltwise_params::GetParamsKey() const {
 }
 
 Datatype EltwiseKernelBase::GetAccumulatorType(const eltwise_params &params) const {
+    // NOTE: 'Workaround' discussed with W. Paramuzov of not promoting shift operations. Not sure what should happen
```

Comment:
I don't think it is good to mention names in comments. Please consider to remove it and leave technical details.

Reply:
Fixed.